.circleci/config.yml

```diff
@@ -115,6 +118,18 @@ jobs:
       - run: ./.circleci/set-aws-credentials.sh
       - run: ./.circleci/deploy-testnet.sh
 
+  ganache_related_tests:
+    machine: true
+    working_directory: /home/circleci/.go_workspace/src/github.com/orbs-network/orbs-network-go/
+    steps:
+      - checkout
+      - run: ./.circleci/install-go.sh
```

Comment:
Why is it more preferential to install go onto machine instead of running inside the golang docker container like we do in any other step?

Reply:
We still run a compose file in this workflow so I need a machine.
The next step for this job would be to check if CircleCI supports passing some flags to the side-containers
If that is possible - I will move this job out of a VM executor completely